add docker network documentation (#1297)
demisto/content-docs: docs/integrations/unit-testing.md

Unit tests should be written in a separate Python file named: `<your_choice>_test.py`. Within the unit test file, each unit test function should be named: `test_<your name>`. More information on writing unit tests and their format is available at the [PyTest Docs](https://docs.pytest.org/en/latest/contents.html). Good place to see example unit tests: [Proofpoint TAP v2 integration](https://github.com/demisto/content/blob/master/Packs/ProofpointTAP/Integrations/ProofpointTAP_v2/ProofpointTAP_v2_test.py)

### Docker network 

> By default, unit-tests are not running with docker network, meaning network is disabled within the container that runs the unit-tests.

Reviewer: ```suggestion
By default, unit-tests are not running with a Docker network, meaning the network is disabled within the container that runs the unit-tests.
```

> By default, unit-tests are not running with access to the network; the network is disabled within the container that runs the unit-tests.

Reviewer 2: ```suggestion
By default, unit-tests are not run with access to the network; the network is disabled within the container that runs the unit-tests.
```

> By default, unit-tests are not running with a Docker network, meaning the network is disabled within the container that runs the unit-tests.

Reviewer 2: ```suggestion
By default, unit-tests are not running with access to the network; the network is disabled within the container that runs the unit-tests.
```

> Refer [here](../packs/packs-format.md#pack-ignore) if the script/integration needs docker network 

Reviewer 2: ```suggestion
Refer to [here](../packs/packs-format.md#pack-ignore) if the script/integration tests need access to the network. 
```
